ItsME/Presentation/Scenes/TotalCV/CVEditing/CoverLetterEditing/CoverLetterEditingViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  CoverLetterEditingViewController.swift
+//  ItsME
+//
+//  Created by MacBook Air on 2023/03/03.
+//
+
+import UIKit
+import Then
+import SnapKit
+
+class CoverLetterEditingViewController: UIViewController, UITableViewDataSource {
```

Comment:
해당 클래스가 상속될 여지가 없으면 final 을 붙여주는게 성능상의 이점이 있습니다.
```suggestion
final class CoverLetterEditingViewController: UIViewController, UITableViewDataSource {
```

Reply:
그리고 DataSource 나 Delegate 프로토콜은 extension 을 활용해서 분리하면 가독성이 더 나아질 것 같습니다.